catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/QueryResponseImpl.java

```diff
@@ -50,6 +50,8 @@
 
     List<Result> resultList = null;
 
+    private long timeout;
```

Comment:
@pklinef, they are set explicitly as a TimeUnit below. Do we really want 'timeoutMillis, TimeUnit.MILLISECONDS`?